src/frontend/apps/desk/src/i18n/translations.json

```diff
@@ -56,15 +56,24 @@
       "Last update at": "Dernière modification le",
       "People": "People",
       "People Description": "Description de People",
-      "404 - Page not found": "404 - Page introuvable",
+      "404 title": "Ooops !",
+      "404 paragraph": "Il semble que la page que vous cherchez n’existe pas ou ne peut pas  s’afficher correctement.",
       "Something bad happens, please retry": "Une erreur inattendue s'est produite, rechargez la page.",
       "Panel create new team": "Panneau de création d'un nouveau groupe",
       "icon group": "icône groupe",
       "Name the team": "Nommer le groupe",
       "Team name": "Nom du groupe",
       "Cancel": "Annuler",
       "Create the team": "Créer le groupe",
-      "Create a new team": "Créer un nouveau groupe"
+      "Create a new team": "Créer un nouveau groupe",
+      "back to home page": "Retourner à la page d'accueil"
+    }
+  },
+  "en": {
+    "translation": {
+      "404 title": "Ouch !",
+      "404 paragraph": "It seems that the page you are looking for does not exist or cannot be displayed correctly.",
+      "back to home page": "Back to home page"
```

Comment:
To translate we pass by a sass named `Crowdin`, I have to show you how it is working.